templates/Web/Pages/React.List/.template.config/template.json

```diff
@@ -12,7 +12,7 @@
     "wts.platform": "Web",
     "wts.projecttype": "FullStackWebApp",
     "wts.frontendframework": "ReactJS",
-    "wts.backendframework": "NodeJS",
+    "wts.backendframework": "all",
```

Comment:
Do we need this change here as it works for blank page only and this is React List?

Reply:
We need this so that React can generate a list template for all backend frameworks and not just NodeJS. Otherwise, if you select a list framework with a Flask application, it gives you an error that templates cannot be generated.